eng/pipelines/test-integration-job.yml

```diff
@@ -20,7 +20,11 @@ parameters:
 
 steps:
   - ${{ if eq(parameters.shallowCheckout, true) }}:
-    - template: checkout-task.yml
+    - template: checkout-windows-task.yml
+  - ${{ if ne(parameters.shallowCheckout, true) }}:
```

Comment:
does azure-pipelines-integration.yml need to be updated?

Reply:
`azure-pipelines-integration.yml` imports `eng/pipelines/test-integration-job.yml` as a template so I don't think so?

Reply:
oops, should have looked above the fold :)